virttest/utils_test/__init__.py

```diff
@@ -2685,6 +2687,9 @@ def setup_ssh_auth(self, vm_ip, vm_pwd, vm_user="root",
             elif index == 3:
                 logging.debug("Logged in now...")
                 break
+            elif index in [4, 5]:
```

Comment:
we should be handling carefully for the real error that can be expected in the if conditions and break, IIUC the existing code would anyway ignore the warning and continue to the loop as it doesn't enter any conditions

Reply:
@balamuruhans , In my previous test, the problem came to me that the '/usr/bin/ssh-copy-id' and 'warning xxx' can not be bypassed and failed to login the VM.  But I can not reproduce it now. So I would like to close it for now. And if it happens again, I will open it. Thank you.